fitbenchmarking/fitbenchmarking
# Renames SasView controller to Bumps controller

#### Description of Work

Fixes #536 

The `sasview` controller called bumps underneath, this PR changes the naming convention to use `bumps` instead.

#### Testing Instructions

1. Check that `sasview` is only mentioned in the parsers and that all tests run

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
